express-mutations/server.ts

```diff
@@ -0,0 +1,114 @@
+import express from 'express';
+import pg from 'pg';
+import { ClientError, errorMiddleware } from './lib/index.js';
+
+const db = new pg.Pool({
+  connectionString: 'postgres://dev:dev@localhost/pagila',
+  ssl: {
+    // Allow non-SSL traffic to localhost
+    rejectUnauthorized: false,
+  },
+});
+
+const app = express();
+
+app.use(express.json());
+
+app.post('/api/actors', async (req, res, next) => {
+  try {
+    const sql = `
+  insert into "actors" ("firstName", "lastName")
+  values ('Brendan', 'Eich')
+  returning *
+  `;
+    const results = await db.query(sql);
+    const post = results.rows;
+    if (!post) {
+      throw new ClientError(404, 'It is missing either firstName or lastName');
```

Comment:
You don't return a 404 from a POST. If firstName or lastName are missing you would return a 400, but that check needs to be made up above, before running the SQL query.